CMakeLists.txt

```diff
@@ -116,6 +116,8 @@ file(GLOB_RECURSE RELAY_SRCS
     )
```

Comment:
add testcases -- you can add the posit functions right into the runtime so that they're accessible. just give them good names.